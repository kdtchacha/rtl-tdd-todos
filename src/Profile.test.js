import React from "react";
import { render } from "@testing-library/react";
import Profile from "./Profile";

describe("<Profile/>", () => {
    it('shows the props correctly', () => {
        const utils = render(<Profile username="chacha" 
        name='이가희' />);
        utils.getByText('chacha');
    })
    it('matches snapshot', () => {
        const utils = render(<Profile username="chacha" name="이가희" />);
        expect(utils.container).toMatchSnapshot();
    })
})
