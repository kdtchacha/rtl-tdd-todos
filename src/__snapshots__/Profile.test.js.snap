// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Profile/> matches snapshot 1`] = `
<div>
  <div>
    <b>
      chacha
    </b>
      
    <span>
      (
      이가희
      )
    </span>
  </div>
</div>
`;
